contents/customers/contra.md

```diff
@@ -0,0 +1,54 @@
+---
+title: How Contra used session recordings to improve registrations by 30%
+customer: Contra
+logo: ../images/customers/contra/contra_logo.png
+featuredImage: ../images/customers/contra/contra_featured.png
+industries:
+    - Saas
+users:
+    - Product
+    - Engineering
+toolsUsed:
+    - Feature Flags
+    - Session Recording
+    - Product Analytics
+---
+
+Contra is a freelance marketplace for service independent creatives and engineers. It empowers its community to find work that fits the way they want to live. Since 2022 [Contra](http://www.contra.com), has used PostHog Cloud to power many of its engineering and product decisions, though previously the stack was a lot more complex.
+
+“Originally we were using multiple platforms,” said Allison Nulty, Head of Product at Contra. “That meant Mixpanel for analytics, LaunchDarkly for feature flags, Segment for front-end tracking and FullStory for recordings. In isolation they could work well, but they were costly even then.”
+
+As a result of spreading functionality across multiple tools, Contra also faced data inconsistencies. Tools such as Segment were helpful for forwarding data between platforms, but also added to the costs and upkeep required. Eventually, Contra decided to consolidate the stack as much as possible — and that’s where PostHog came in. 
```

Comment:
One note is that we currently still use Segment and are happy with the service - I might add that Segment was useful for tracking _and_ forwrding

Reply:
```suggestion
As a result of spreading functionality across multiple tools, Contra also faced data inconsistencies. Tools such as Segment were helpful tracking for forwarding data between platforms, but also added to the costs and upkeep required. Eventually, Contra decided to consolidate the stack as much as possible — and that’s where PostHog came in. 
```